--- Linux-GNU.cmake	2013-06-03 14:24:05.850206868 +0600
+++ Linux-GNU.cmake.new	2013-06-03 14:22:08.000000000 +0600
@@ -21,5 +21,5 @@
 macro(__linux_compiler_gnu lang)
   # We pass this for historical reasons.  Projects may have
   # executables that use dlopen but do not set ENABLE_EXPORTS.
-  set(CMAKE_SHARED_LIBRARY_LINK_${lang}_FLAGS "-rdynamic")
+  #set(CMAKE_SHARED_LIBRARY_LINK_${lang}_FLAGS "-rdynamic")
 endmacro()
